File: u1-becoming-a-developer/d03-javascript/student/star-wars/ig_88a.js
while(true){
  var user = prompt("Freeze! Who are you?");
  if (user.toLowerCase() === "zap"){
    for (var i = 0; i < 20; i++){
      window.alert("zap");
    }
    break;
  } else if(user.toLowerCase() !== "han solo"){
    window.alert("You must die! (zap zap kaboom)");
  } else if(user.toLowerCase() === "han solo"){
    window.alert("You are mine at last, Solo!");
    for (var i = 0; i < 20; i++){
      window.alert("zap");
    }
    var percent = (Math.random() * 100)
    if (percent <= 50) {
      window.alert("Solo is caught! Jabba the Hutt will pay handsomely...");
      break;
    } else {
      window.alert("I'll get you next time, Solo!");
    }
  }
  }

File: u4-ruby-on-rails/d17-ruby-slippers/student/homework/starwars/ig_88a.rb
solo = true
while solo do
  puts "Freeze! Who are you?"
  user = gets.chomp.downcase
  if user == 'zap'
    for x in 0...20
      puts "zap"
    end
  elsif user != 'han solo'
    puts "You must die! (zap zap kaboom)"
  elsif user == 'han solo'
    puts "You are mine at last, Solo!"
    for y in 0...20
      puts "zap"
    end
    percent = rand(100)
    if percent <= 50
      puts "Solo is caught! Jabba the Hutt will pay handsomely..."
    else
      puts "I'll get you next time, Solo!"
    end
    solo = false
  end
end
